nyxx_interactions/lib/src/Interactions.dart

```diff
@@ -25,6 +22,12 @@ class Interactions {
   final _buttonHandlers = <String, ButtonInteractionHandler>{};
   final _multiselectHandlers = <String, MultiselectInteractionHandler>{};
 
+  /// Commands registered by bot
+  Iterable<SlashCommand> get commands => UnmodifiableListView(this._commands);
+
+  /// Reference to client
+  final Nyxx client;
```

Comment:
```suggestion
  final Nyxx _client;
```
Use a private variable as the user shouldn't need to access this